content/06-sqlite/06-update-data.mdx

```diff
@@ -154,3 +154,75 @@ SELECT * FROM author;
 ## Conclusion
 
 In this guide, we took a look at the basic ways that you can modify existing data within a table using the `UPDATE` command. Execution of these basic concepts allows you to specify the exact criteria necessary to identify the existing rows within a table, update column names with values, and optionally return the rows that were impacted with `RETURNING`. The `UPDATE` command is critical for managing your data after the initial insertion into your databases.
+
+## FAQ
+
+<details><summary>How do you perform a bulk update in SQLite?</summary>
+
+In SQLite, you can use the `UPDATE` command looking something like the following:
+
+```
+UPDATE my_table
+SET
+        column1 = value1,
+        column2 = value2,
+WHERE
+        id = 1;
```

Comment:
Since this is a response about bulk updating, it may make sense to use an example that is more likely to return multiple results.  Even something silly like `WHERE color = 'blue'` could be helpful.